templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
+This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.
+
+We would like to put some more effort in bringing up some amazing talks to be presented this  Pycon 2020. We are creating a team of volunteers to review the talk proposals that should be presented in Pycon. Since this is a peer review process and driven by volunteers, we are looking for more volunteers to join us and help in making this a success. We believe you have great experience in Python and/or Python community and we would love if you can be a part of this group.
+
+Your role will be mainly to review the content of the talks submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide your feedback if this should be a part of Pycon 2020. 
+
+If you think you can help us with this, please reply to the email confirming and we will add you to the group where we will discuss and plan/schedule sessions. 
+FAQs
+1. How much time do I need to dedicate ?
+Ans: For review, it will be around 10-15 hours over a period of 20 - 25 days
```

Comment:
Have to let them know they are needed and are extremely essential to process.

```suggestion
If you feel you can help and play an essential role in growing our community's beloved conference, please reply to the email confirming your participation and we will add you to the group where we can further discuss and plan/schedule sessions. 

FAQs

1. How much time would I need to commit?
For review, it will be around 10-15 hours over a period of 20 - 25 days
```

Wouldn't the time be more, I mean I am no reviewer. PyCon does get a lot of proposals + we have this new video feature. @sayanchowdhury what do you think

Reply:
Got that details from ritesh, Yes a confirmation would be nice.

Reply:
Confirmed with the team we will go with buffer of 20 hours.